lib/init.ts

```diff
@@ -71,4 +71,8 @@ export function init(options: InitParams) {
   // user agent
   this._ua = DEFAULT_ALGOLIA_AGENT;
   this._uaURIEncoded = encodeURIComponent(DEFAULT_ALGOLIA_AGENT);
+
+  if (supportsCookies()) {
+    this.setUserToken(this.ANONYMOUS_USER_TOKEN);
+  }
```

Comment:
Since we're not automatically calling setUserToken on load, we should throw an error when user has forgotten to set it.

Reply:
It might be best to delegate that task to sendEvent

Reply:
Oh I see. So we're supposed to throw an error when user hasn't set any user token, for example, in node env?
Setting user token is mandatory before `sendEvent`?

Reply:
yes there must be no situation where an event can fire with no userToken in the payload.
When the cookie is enabled, this never happens because the anonymous token is always serving as a fallback.
On node, this part is gone. We need to alert the user that they need to either add userToken to their event, or set it with  `setUserToken`

Reply:
https://github.com/algolia/search-insights.js/pull/141/commits/6e0f266cbc56d9df7f641cae832407e79f71f83d